python4j/python4j-core/src/main/java/org/eclipse/python4j/PythonTypes.java

```diff
@@ -0,0 +1,352 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Konduit K.K.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.python4j;
+
+
+import org.bytedeco.cpython.PyObject;
+
+import java.util.*;
+
+import static org.bytedeco.cpython.global.python.*;
+import static org.bytedeco.cpython.global.python.Py_DecRef;
+
+public class PythonTypes {
+
+
+    private static List<PythonType> getPrimitiveTypes(){
+        return Arrays.asList((PythonType)
+                STR,
+                INT,
+                FLOAT,
+                BOOL
+        );
```

Comment:
Not a big deal, but FYI this is something we can also do: `Arrays.<PythonType>asList(` when java complains about types